Corzbank.Services/TransferService.cs

```diff
@@ -64,20 +65,20 @@ public async Task<TransferDTO> CreateTransfer(TransferModel transferRequest)
                 transferRequest.ReceiverPhoneNumber = null;
 
                 if (transferRequest.ReceiverCardNumber == null)
```

Comment:
The better approach of checking strings on null is to use string.IsNullOrWhiteSpace, because it will also check for white spaces. Refactor in current file